pandas/io/json/_json.py

```diff
@@ -718,6 +733,15 @@ def _combine_lines(self, lines) -> str:
         lines = filter(None, map(lambda x: x.strip(), lines))
         return "[" + ",".join(lines) + "]"
 
+    def _jsonstring_to_list_generaor(self, data: str) -> Iterator[str]:
+        prev_index = -1
```

Comment:
can you add doc-string here.

Reply:
removed the function entirely so not needed now